src/plugins/auto/plugin.cpp

```diff
@@ -266,12 +266,6 @@ InferenceEngine::Parameter MultiDeviceInferencePlugin::GetMetric(const std::stri
     }
 }
 
-// Is called only when caching is enabled
-IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetwork(const std::string& modelPath,
-                                                                        const std::map<std::string, std::string>& config) {
-    return LoadNetworkImpl(modelPath, {}, config);
-}
-
 IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(const CNNNetwork &network,
```

Comment:
@myshevts maxim, can you help double confirm if this is still needed?
From my understanding, there is loadfromcache implemented both for loadnetwork with modelpath & cnnNetwork.
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/ie_core.cpp#L693

Reply:
these lines were added by @nosovmik , so let's ask him

Reply:
ah..I suppose the cache optimize is for 2 parts, part A:  modelfile->CNNnetwork and part B: load from cache (modelfile and cnn network both supported).  with these lines, it's expected to support partA + part B, while without , only part B is supported, is my understanding right? @nosovmik